assets/messages.yml

```diff
@@ -656,10 +656,22 @@ messages:
       shortcut: min
       message: |-
         *Thank you for your report!* 
-        However, this issue is {color:red}*Invalid*{color}.
+        However, this issue is {color:#FF5722}*Invalid*{color}.
```

Comment:
Since you have changed this here as well, could you please also change it for `outdated-client`? (Or maybe we should do that in a separate pull request to keep these changes separate.)